DOCS/man/options.rst

```diff
@@ -5314,17 +5313,17 @@ them.
 
 ``--cscale=<filter>``
     As ``--scale``, but for interpolating chroma information. If the image is
-    not subsampled, this option is ignored entirely.
+    not subsampled, this option is ignored entirely. If this option is unset,
+    the filter implied by ``--scale`` will be applied.
 
 ``--dscale=<filter>``
-    Like ``--scale``, but apply these filters on downscaling instead. If this
-    option is unset, the filter implied by ``--scale`` will be applied.
+    Like ``--scale``, but apply these filters on downscaling instead.
```

Comment:
Now we set `dscale` to `mitchell` as default, so it will not be copied from `scale`. Unless someone explicitly do `--dscale=""`, but it is corner case and would be confusing in docs.